### DISCUSSION TITLE
1.0 Release

### DISCUSSION THREAD

**Discussion by danwos** [useblocks/sphinx-needs#562] [General]
Hi, all Sphinx-Needs friends,

I'm planning a Sphinx-Needs 1.0 change, which has some breaking API changes.

Mainly ``sphinxcontrib.needs`` is moved to ``sphinx_needs``. 
This is primarily because of bugs with ``pip install -e`` and the ``sphinxcontrib.needs`` package structure.
In most cases, you could not install Sphinx-Needs in "develop" mode, which makes it hard to debug.
With the new structure, this is now possible and I have the feeling that also the poetry dependency resolver is now much faster.

However, as ``sphinxcontrib.needs`` is history with ``1.0``, I want your thoughts about renaming:
- the repository to ``useblocks/sphinx-needs``
- the RTD documentation URL to `sphinx-needs.readthedocs.io`

Any thoughts on this?

**Comment by StephanKuempel** [comment]
So any dependant extension/tool using sphinxcontrib.needs.api (for e.g. to create needs with add_node) needs to adapt to using sphinx_needs.api then, correct ?

**Reply by danwos** [reply]
Correct. 
This must be changed in all tools.

**Comment by twodrops** [comment]
To be consistent, we could take the chance and rename it to `useblocks/sphinx-needs`.

How about renaming the repo to `useblocks/sphinx-needs` and directly creating a new repo as `useblocks/sphinxcontrib.needs` with a readme mentioning the new repo URL and info on how to change the git origin to point to the new repo?
Maybe also for `sphinxcontrib-needs.readthedocs.io`.

What do you think?

**Reply by danwos** [reply]
Yes I agree, if we rename everything the old URLs should redirect are at least show a big, red banner :)

I'm not sure if GitHub does automatically a redirect. We will find out. But if not, I like your idea to directly recreate an empty repo under the same name and just add a README.
